sklearn/feature_extraction/_hashing_fast.pyx

```diff
@@ -89,7 +89,7 @@ def transform(raw_X, Py_ssize_t n_features, dtype,
     indptr_a = np.frombuffer(indptr, dtype=indices_np_dtype)
 
     if indptr[-1] > np.iinfo(np.int32).max:  # = 2**31 - 1
-        if sp_version < (0, 14):
```

Comment:
The minimum Scipy supported version is 0.19.1 this should be removed ideally in another PR.

Can you revert this change?

I'll open an issue about his.

Reply:
OK.